.github/workflows/dependency-check.yml

```
@@ -0,0 +1,70 @@
+name: dependency check
+
+on:
+  workflow_call:
+    secrets:
+      GH_ACCESS_TOKEN:
+        required: true
+      AWS_MAVEN_ACCESS_KEY_ID:
+        required: true
+      AWS_MAVEN_ACCESS_SECRET_KEY:
+        required: true
+
+jobs:
+  dependency-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: project
+
+      - name: Clone gradle-scripts
+        uses: actions/checkout@v3
+        with:
+          repository: tastyworks/gradle-scripts
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          path: gradle-scripts
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: 'temurin'
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x ./project/gradlew
+
+      - name: Cache Gradle
+        uses: actions/cache@v3
+        with:
+          path: ~/.gradle
+          key: ${{ runner.os }}-gradle-${{ hashFiles('project/*.gradle', 'project/gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Run Gradle dependencyCheckAnalyze
+        working-directory: ./project
+        run: >
+          ./gradlew
+          -PAWS_MAVEN_ACCESS_KEY_ID=${{ secrets.AWS_MAVEN_ACCESS_KEY_ID }}
+          -PAWS_MAVEN_ACCESS_SECRET_KEY=${{ secrets.AWS_MAVEN_ACCESS_SECRET_KEY }}
+          dependencyCheckAnalyze --info
+
+      - name: Upload dependency check report
+        uses: actions/upload-artifact@v3
+        with:
+          name: dependency-check-report
+          path: "${{ github.workspace }}/project/build/reports/dependency-check-report.html"
+          if-no-files-found: error
+
+      - name: Send link to slack
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
+        env:
+          SLACK_WEBHOOK_URL: "${{ env.BOULANGERIE_CVE_SLACK_WEBHOOK }}"
```

This is actually scope as a `vars`

```suggestion
          SLACK_WEBHOOK_URL: "${{ vars.BOULANGERIE_CVE_SLACK_WEBHOOK }}"
```